From 7f73fbe4e6863e50c0fb1519def48bd83c2fdaf3 Mon Sep 17 00:00:00 2001
From: Niek van der Maas <mail@niekvandermaas.nl>
Date: Fri, 23 Oct 2020 15:06:18 +0200
Subject: [PATCH] Disable props

---
 products/anbox.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/products/anbox.mk b/products/anbox.mk
index 6ab2330..43d8556 100644
--- a/products/anbox.mk
+++ b/products/anbox.mk
@@ -68,7 +68,7 @@ PRODUCT_PACKAGES += \
 PRODUCT_PROPERTY_OVERRIDES += \
 	ro.hardware=goldfish \
 	ro.hardware.hwcomposer=anbox \
-	ro.kernel.qemu.gles=1 \
+	ro.kernel.qemu.gles=0 \
 	ro.kernel.qemu=1
 	ro.adb.qemud=1
 
-- 
2.28.0

